tasks/9_init.yml

```diff
@@ -9,25 +9,35 @@
   when:         ansible_os_family == "RedHat"
   notify:       restart_seafile
 
-- name:         provision init script
+- name:         provision sysvinit script
   template:
     src:        init/{{ item.name }}.initd.{{ ansible_os_family }}
     dest:       /etc/init.d/{{ item.name }}
     owner:      root
     group:      root
     mode:       0750
-  when:         ansible_os_family in item.os_family
+  when:         ansible_service_mgr == "sysvinit"
```

Comment:
I read the https://manual.seafile.com/deploy/start_seafile_at_system_bootup.html and now I understand that RedHat requires to scripts and Debian only one. I updated my pull request.